docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-register-solver-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-register-solver-proc']
+= Register the Solver using the REST API 
+
+Once the Solver has been created, you can register it using the REST API. By registering the Solver, you can then submit planning problems in the request body to find the optimal solution.
+
+.Prerequisite
+
+* The employee rostering project has been set up and deployed according to the previous chapters in this guide.
+* The Solver has been configured as instructed in <<wb-employee-rostering-solver-configuration-proc>> and <<wb-employee-rostering-solver-termination-configuration-proc>>.
+* You have successfully built and deployed the employee rostering project.
+
+.Procedure 
+. Create a HTTP request using the following header:
++
+[source]
+----
+authorization: Basic ZG1BZG1phjphZG1pbg==
+X-KIE-ContentType: xstream
+content-type: application/xml 
+----
+. Register the Solver using the following request:
+PUT::
+`http://localhost:8080/kie-server/services/rest/server/containers/employeerostering_1.0.0-SNAPSHOT/solvers/EmployeeRosteringSolverConfig`
```

Comment:
Let's use a different name for the solver than the name of the solver config. Firstly, the name can be arbitrary, secondly, the type of the resource this creates is a solver, not a config, so the "Config" part of the name doesn't make sense. Good names (from worse to better):
- solver1
- employee-rostering-solver-1
- EmployeeRosteringSolver